src/client/components/createBtn.js

```diff
@@ -18,6 +19,8 @@ class CreateBtn extends React.Component {
       owner: '',
       priority: '',
     };
+
+    axios.post('/api/tabledatas', newRow);
     onClick(newRow);
```

Comment:
If the `post` for `/api/tabledatas` failed, should this `onClick` be triggered?

Reply:
I think it still would be triggered because although I get 404 error, it still create a new row

Reply:
Yes, so the next question would be, when we fail to request, should we make the action applied into UI? If not, maybe you can try to think about how to avoid this.

Reply:
OK, I think I know your meaning. We also need to stop UI to change once we  fail to request.